custom_components/hubitat/hubitatmaker/hub.py

```diff
@@ -462,7 +466,7 @@ async def _start_server(self) -> None:
         # machine and the Hubitat hub are on the same network.
         with _open_socket(socket.AF_INET, socket.SOCK_DGRAM) as s:
             s.connect((self.host, 80))
-            address = s.getsockname()[0]
+            address = "0.0.0.0"
```

Comment:
This will cause the event POST URL in the Maker API to be http://0.0.0.0:xxx if the user doesn't manually specify their event server URL in the integration options.